ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -108,6 +108,23 @@ def cancel_timeout(self):
             self.timeout_thr = None
 
     def copp_test(self, packet, count, send_intf, recv_intf):
+        '''
+        Pre-send some packets for a second to absorb the CBS capacity.
+        '''
+        if self.needAbsorbCBS:
+            timeout_start = time.time()
+            sendInFirst = 0
+            while time.time() < timeout_start + 1:
+                testutils.send_packet(self, send_intf, packet)
+                sendInFirst += 1
+            '''
+            count_matched_packets() takes a long time (more than 1 second)
+            Suspend counting matched packetes till all testing packets are sent.
+            Copy the received packets to 'queue'
+            '''
+            queue = self.dataplane.packet_queues[(recv_intf[0],recv_intf[1])]
```

Comment:
I think you don't need to read anything. Just flush dataplane after 1-2 seconds of sending.

Reply:
Yes. If count_matched_packets() can do a short time. I removed the related code.